Source/FactionColonies/util/SimulateBattleFC.cs

```diff
@@ -372,7 +373,8 @@ public static void resetPlayerColonyRelations()
                     PCFaction.TryAffectGoodwillWith(faction,
                         (Find.FactionManager.OfPlayer.RelationWith(faction).baseGoodwill -
                          PCFaction.RelationWith(faction).baseGoodwill));
-                    PCFaction.TrySetRelationKind(faction, Find.FactionManager.OfPlayer.RelationKindWith(faction));
+                    // FIXME Workaround, since method TrySetRelationKind is gone
+                    PCFaction.SetRelationDirect(faction, Find.FactionManager.OfPlayer.RelationKindWith(faction));
```

Comment:
Request for Comment:
This is currently only a workaround. I opened a Thread on the forums for further Help:
https://ludeon.com/forums/index.php?topic=54667.0